.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# these owners will be requested for review when someone
+# opens a pull request.
+*       @dav3r @felddy @hillaryj @jsf9k @mcdonnnj @cisagov/team-ois
```

Comment:
This list should be whittled down to only those folks (at least two) who are actually familiar with this repo.  I'm not sure who is in that list, but I know I am not.

Reply:
Also, leave @cisagov/team-ois off the list.  I don't think we want to do that anymore.  I created cisagov/skeleton-generic#50 to remove that, so it is addressed and percolates down the next time we have to release the Kraken for more substantive changes.

Reply:
Updated in https://github.com/cisagov/assessment-data-import-terraform/pull/4/commits/2bab763073ca17d6f43ebbcb5910254a47647ed8